### PR TITLE
Add handling of duplicate anims and non-64x64 anim variants to download script

### PR DESCRIPTION
**Pull request by AndyMender**
Addresses: https://github.com/Tuxemon/Tuxemon/issues/1314

Just a note that this still doesn't catch all corner cases like:
- https://wiki.tuxemon.org/File:Firelion_right.gif (the smaller sized anim is 64x61, not 64x64)
- https://wiki.tuxemon.org/File:Rockfall_193.gif (none of the anims are 64x64)

### CODE REVIEW

**Comment by AndyMender** [comment]
@kerizane I think this PR covers what we discussed in the linked ticket. Are there any changes required? :)

**Comment by kerizane** [comment]
Sorry, I know I suggested this implementation, but I think we should do it this way instead:

https://github.com/Tuxemon/Tuxemon/issues/1314#issuecomment-1249730021

If the animations are currently broken, then I will accept a PR that fixes the animations without updating this script.

File: scripts/download_technique_animations.py [hunk 1]
```diff
@@ -87,24 +89,47 @@ def download_animation_credits(gif_page_url: str) -> str:
...
                 ANIMATION_DIR, f"{base_name}{frame:02}.png"
             )
+
+            gif_image.seek(frame)
+            # Check if animation is not a duplicate or a non-64x64 variant on an existing one
+            if os.path.isfile(frame_filename):
+                with Image.open(frame_filename) as frame_image:
+                    if not ImageChops.difference(
+                        gif_image, frame_image
+                    ).getbbox():
```
Comment (kerizane) [inline scripts/download_technique_animations.py:118]: What does this if statement check?

Comment (AndyMender) [reply scripts/download_technique_animations.py:118]: It compares 2 image objects. I should've put a comment above to make it clearer, apologies!

**Comment by AndyMender** [comment]
> Sorry, I know I suggested this implementation, but I think we should do it this way instead:
> 
> [#1314 (comment)](https://github.com/Tuxemon/Tuxemon/issues/1314#issuecomment-1249730021)
> 
> If the animations are currently broken, then I will accept a PR that fixes the animations without updating this script.

I'll keep the branch just in case, but yeah, if the preferred way is having a strict category for used animations, I will need to implement that differently.

**Pull request closed without merging**